massdriver-application-azure-app-service/_variables.tf

```diff
@@ -0,0 +1,45 @@
+variable "repo" {
+  type = object({
+    docker_image     = string
+    docker_image_tag = string
+  })
+}
+
+variable "dns" {
+  type = object({
+    enable_dns          = bool
+    txt_record          = string
+    cname_record        = string
+    zone_name           = string
+    zone_resource_group = string
+  })
+}
+
+variable "application" {
+  type = object({
+    location             = string
+    sku_name             = string
+    minimum_worker_count = number
+    maximum_worker_count = number
+    notification_email   = string
+  })
+}
+
+variable "md_metadata" {
+  type = object({
+    name_prefix = string
+  })
+}
+
+variable "name" {
+  type = string
+}
+
+variable "tags" {
+  type = object({
```

Comment:
We may want to do this as an any so that if we start sending more tags/labels in the future it doesn’t break this type signature

Reply:
By doing that, would I also remove the strings below? Just leave it as `type = any`?

Reply:
Yep!